README.md

```diff
@@ -1 +1,160 @@
-# Budget_App
\ No newline at end of file
+
+
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h2><b>SpendWise</b></h2>
+<br>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget Manager ](#-budget-manager-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Test](#test)
+  - [Authors ](#authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 Budget Manager <a name="about-project"></a>
+
+SpendWise a mobile web application where user can manage his/her budget: user have a list of transactions associated with a category, so that user can see how much money he/she spent and on what.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://ruby-doc.org/3.2.2/">Ruby</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **List of transactions associated with a category**
+
+- **User can see how much money spent and on what**
+
+## Live Demo
+
+- [Live Demo](https://spendwise-jolr.onrender.com/)
+- [Project video Link](https://www.loom.com/share/310412cfee864f67b113a641be9b30c5?sid=e5ca893a-648c-4323-9531-e592d1739623)
```

Comment:
Good job recording your video presentation but you missed out some requirements concerning your video presentation. Your video was 3mins long instead of 5min and then you never showed us a piece of code and a feature you are most proud of in the project. It would be nice if you recorded another video based on the following requirements:
Video presentation

Provide feedback about the video presentation in the code review summary.

- The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.